core/src/main/java/io/aiven/kafka/tieredstorage/config/RemoteStorageManagerConfig.java

```diff
@@ -178,6 +187,14 @@ public class RemoteStorageManagerConfig extends AbstractConfig {
                     Sensor.RecordingLevel.TRACE.toString()),
                 ConfigDef.Importance.LOW,
                 METRICS_RECORDING_LEVEL_DOC);
+
+        CONFIG
```

Comment:
Should we keep the formatting consistent?

Reply:
Agree, fixing it on this and above metric configs.